docs/sources/explore/logs-integration.md

```diff
@@ -130,38 +129,35 @@ You may encounter long lines of text that make it difficult to read and analyze
 
-If you switch from a Prometheus query to a logs query (you can do a split first to have your metrics and logs side by side) then it will keep the labels from your query that exist in the logs and use those to query the log streams. For example, the following Prometheus query:
+## Live tailing
 
-`grafana_alerting_active_alerts{job="grafana"}`
+To view real-time logs from supported data sources, you can leverage the Live tailing feature in Explore.
 
-after switching to the Logs data source, the query changes to:
+1. Click the **Live** button in the Explore toolbar to switch to Live tail view.
+2. While in Live tail view, new logs will appear from the bottom of the screen, and they will have a fading contrasting background, allowing you to easily track what's new.
+3. If you wish to pause the Live tailing and explore previous logs without any interruptions, you can do so by clicking the **Pause** button or simply scrolling through the logs view.
+4. To clear the view and remove all logs from the display, click on the **Clear logs** button. This action will reset the log view and provide you with a clean slate to continue your log analysis.
+5. To resume Live tailing and continue viewing real-time logs, click the **Resume** button.
+6. If you want to exit Live tailing and return to the standard Explore view, click the **Stop** button.
 
-`{job="grafana"}`
+Using the Live tailing feature, you can keep a close eye on the latest logs as they come in, making it easier to monitor real-time events and detect issues promptly.
 
-This will return a chunk of logs in the selected time range that can be grepped/text searched.
+{{< video-embed src="/static/img/docs/v95/explore_live_tailing.mp4" >}}
 
 ### Logs sample
 
 If the selected data source implements logs sample, and supports both log and metric queries, then for metric queries you will be able to automatically see samples of log lines that contributed to visualized metrics. This feature is currently supported by Loki data sources.
 
-### Live tailing
-
-Use the Live tailing feature to see real-time logs on supported data sources.
-
-Click the **Live** button in the Explore toolbar to switch to Live tail view.
+### Switch from metrics to logs
 
-While in Live tail view new logs will come from the bottom of the screen and will have fading contrasting background so you can keep track of what is new. Click the **Pause** button or scroll the logs view to pause the Live tailing and explore previous logs without interruption. Click **Resume** button to resume the Live tailing or click **Stop** button to exit Live tailing and go back to standard Explore view.
+If you are coming from a metrics data source that implements `DataSourceWithQueryExportSupport` (such as Prometheus) to logging data source that supports `DataSourceWithQueryImportSupport` (such as Loki), then it will keep the labels from your query that exist in the logs and use those to query the log streams.
```

Comment:
```suggestion
If you are coming from a metrics data source that implements `DataSourceWithQueryExportSupport` (such as Prometheus) to a logging data source that supports `DataSourceWithQueryImportSupport` (such as Loki), then it will keep the labels from your query that exist in the logs and use those to query the log streams.
```